hcipy/optics/fiber.py

```diff
@@ -205,15 +316,33 @@ def __init__(self, input_grid, fiber_grid, mode, *args, **kwargs):
 		self.projection_matrix = self.fiber_modes.transformation_matrix
 
 	def forward(self, wavefront):
+		'''Forward propagate the light through the fiber array.
+
+		Parameters
+		----------
+		wavefront : Wavefront
+			The incoming wavefront.
+
+		Returns
+		-------
+		Wavefront
+			The complex amplitudes of each fiber.
+		'''
 		res = self.projection_matrix.T.dot(wavefront.electric_field * self.input_grid.weights)
 		return Wavefront(Field(res, self.fiber_grid), wavefront.wavelength)
```

Comment:
This also needs a complex conjugate somewhere. Either in the fiber modes or the input field.